internal/nonkube/common/systemd.go

```diff
@@ -260,3 +272,50 @@ func IsLingeringEnabled(user string) bool {
 	_, err := os.Stat(lingerFile)
 	return err == nil
 }
+
+func NewSystemdGlobal(platform string) (SystemdGlobal, error) {
+
+	return &systemdGobal{
+		getUid:              os.Getuid,
+		command:             exec.Command,
+		rootSystemdBasePath: rootSystemdBasePath,
+		platform:            platform,
+	}, nil
+}
+
+func (sg *systemdGobal) getCmdEnablePodmanSocket() *exec.Cmd {
+	return sg.command("systemctl", "--user", "enable", "--now", "podman.socket")
+}
+
+func (sg *systemdGobal) getCmdStartPodmanSocket() *exec.Cmd {
+	return sg.command("systemctl", "--user", "start", "podman.socket")
+}
+
+func (sg *systemdGobal) getCmdDisablePodmanSocket() *exec.Cmd {
+	return sg.command("systemctl", "--user", "disable", "--now", "podman.socket")
+}
+
+func (sg *systemdGobal) Enable() error {
+	err := sg.getCmdEnablePodmanSocket().Run()
+	if err != nil {
+		return err
+	}
+
+	err = sg.getCmdStartPodmanSocket().Run()
```

Comment:
When `enable --now` is executed, it is already started, so no need to start it again I belive.